internal/controller/services/monitoring/resources/perses.tmpl.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: perses.dev/v1alpha1
+kind: Perses
+metadata:
+  name: data-science-perses
```

Comment:
If we're adding a data-science perses here then what is the difference in the [other pr](https://github.com/den-rgb/opendatahub-operator/blob/bac3303d9ceb4d690abe051faa2511f35195a6ca/internal/controller/services/monitoring/resources/perses.tmpl.yaml).